Source/Turbo Navigator/TurboNavigator.swift

```diff
@@ -75,6 +75,31 @@ public class TurboNavigator {
         guard let controller = controller(for: proposal) else { return }
         hierarchyController.route(controller: controller, proposal: proposal)
     }
+    
+    /// Allows programmatic navigation of external URLs.
+    ///
+    /// - Parameters:
+    ///   - externalURL: the URL to navigate to
+    ///   - via: navigation action
+    public func open(externalURL: URL, _ via: ExternalURLNavigationAction) {
```

Comment:
```suggestion
    public func open(externalURL url: URL, via action: ExternalURLNavigationAction) {
```

If `via` isn't optional then I think we should expose the name in the function.

Reply:
I don't have a strong preference. The reason I removed the label is because of call readability:

```swift
open(anExternalURL, via: .openViaSafariController)
```

vs.

```swift
open(anExternalURL, .openViaSafariController)
```

I don't mind either way. If you still feel the label works, can you commit your own suggestion? (I don't know if Github allows that, if not, I can merge it!)

Reply:
Since we're now using `ExternalURLNavigationAction` in two places (`handle(externalURL:)` and `open(externalURL:_:)`) one of them will have their readability suffer. 

For example, using `.viaSafariViewController` will read well in the former, but not the latter:

```swift
func handle(externalURL: URL) -> ExternalURLNavigationAction {
    .openViaSafariViewController
}
```

```swift
open(externalURL, via: .viaSafariViewController)
```

vs.

```swift
func handle(externalURL: URL) -> ExternalURLNavigationAction {
    .safariViewController
}
```

```swift
open(externalURL, via: .safariViewController)
```

Again, this is such a small change I don't have a strong opinion, so feel free to go with whichever. I wanted to point out though, that both use cases have different grammar.